README.md

```diff
@@ -111,6 +111,7 @@ If you want to push the received data to the store, set this option to `true` an
 import { JSONAPISerializer } from 'ember-custom-actions';
 export default JSONAPISerializer.extend();
 ```
+If you use RESTSerializer instead of JSONAPISerializer to serialize your models, you don't need our extension of JSONAPISerializer.
```

Comment:
I get it. Your `RESTSerializer` is a nice addition. Hope this feature flag will become a feature. :)